### PR TITLE
12 bind table error

### LINKED ISSUE #12: bind_table function error when rowbyvar or tablebyvar arguments are used
When creating the AE table of body system by preferred term, an error was encountered by users in the bind_table function call. The reprex code is below:

library(dplyr)
library(tidytlg)

## Process data
adae <- cdisc_adae %>%
  filter(SAFFL == "Y", TRTEMFL == "Y") %>% 
  rename(TRT01AN = TRTAN)

adsl <- cdisc_adsl %>% 
  filter(SAFFL == "Y")

## Analysis set row
t1 <- adsl%>%
  freq(colvar = 'TRT01AN',
       rowvar = 'SAFFL',
       rowtext = 'Analysis set: Safety',
       statlist = statlist("n"))

## Subjects with 1 or more AEs
t2 <- adae %>%
  freq(denom_df = adsl,
       colvar = 'TRT01AN',
       rowvar ='TRTEMFL',
       rowtext = 'Subjects with 1 or more AEs',
       statlist = statlist("n (x.x%)"), 
       subset =  TRTEMFL == "Y")

## create AE header
t3.1 <- data.frame(label = c("System organ class", "Preferred term"),
                 row_type = "HEADER")

## AE System organ class / Preferred term
t3 <- adae %>%
  nested_freq(denom_df = adsl,
              colvar = 'TRT01AN',
              rowvar = 'AEBODSYS*AEDECOD',
              subset = TRTEMFL == "Y",
              statlist = statlist("n (x.x%)"))

tbl <- bind_table(t1, t2, t3.1, t3,
                  colvar = "TRT01AN",
                  rowbyvar = "AEBODSYS")

**Error: rowbyvar 'AEBODSYS' does not exist in df 1 or is all NA for function: bind_table**

If the rowbyvar argument in the above bind_table call is removed, another error pop up.
tbl <- bind_table(t1, t2, t3.1, t3,
                  colvar = "TRT01AN")

**Error: dfs 3 are missing the required fields for function bind_table**

These errors may be due to the check_args implementation.

### PR DESCRIPTION
**Pull request by elimillera**
Updates the check functions to remove the ones that are failing. I kept them in just in case we want to refine them in the future.

Closes #12

### CODE REVIEW

File: tests/testthat/test-bind_table.R [hunk 1]
```diff
@@ -42,3 +42,89 @@ test_that("format variables are added", {
...
+                colvar = 'TRT01AN',
+                rowvar = 'AEBODSYS*AEDECOD',
+                subset = TRTEMFL == "Y",
+                statlist = statlist("n (x.x%)"),
+                row_header = c("System organ class/Preferred Term"))
+
+  expect_silent({
+    tbl <- bind_table(t1, t2, t3,
+                      colvar = "TRT01AN")
+    #rowbyvar = "AEBODSYS")
```
Comment (shengwei66) [inline tests/testthat/test-bind_table.R:83]: Can we include the rowbyvar = "AEBODSYS" in the bind_table function in this test? It's important to include the rowbyvar argument here, and so the column of AEBODSYS is automatically removed from the tbl, which needs to be done before passing to gentlg function.

**Review by shengwei66 — CHANGES_REQUESTED** [review]
@elimillera the changes look good! Just one small comment of the test at line 83 for including the rowbyvar argument in the bind_table call.

File: tests/testthat/test-bind_table.R (hunk 1 above)
Comment (elimillera) [reply tests/testthat/test-bind_table.R]: Looks like this was a duplicate test with the one below. Updated

**Review by shengwei66 — APPROVED** [review]
@elimillera the test looks good. Thanks.

**Pull request merged**